internal/uvm/wait.go

```diff
@@ -6,14 +6,14 @@ import (
 )
 
 func (uvm *UtilityVM) waitForOutput() {
-	logrus.WithField(logfields.UVMID, uvm.ID()).
+	logrus.WithField(logfields.UVMID, uvm.id).
 		Debug("UVM exited, waiting for output processing to complete")
 	if uvm.outputProcessingDone != nil {
 		<-uvm.outputProcessingDone
 	}
 }
 
-// Waits synchronously waits for a utility VM to terminate.
+// Wait waits synchronously waits for a utility VM to terminate.
```

Comment:
"Wait waits synchronously waits"

Reply:
Sounds like perfect grammar to I

Reply:
Lol I didn't even read it just fix the linter error. Will update